src/components/templates/TemplateWorkflowList.vue

```diff
@@ -0,0 +1,84 @@
+<template>
+  <div class="template-list-view">
+    <DataTable
```

Comment:
Can you make sure the DataTable take the full height so there is no layout shift of paginator when switching to last page?

Reply:
done, see
![image](https://github.com/user-attachments/assets/2f9e64d9-8a93-4471-abad-da00d3cd0dc0)